src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -33,8 +34,7 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
   showNametag,
   onClick,
   showStatus,
-  large,
-  medium,
+  size = "legacy",
```

Comment:
To not break existing uses of `<UserAvatar>` where no size is explicitly set, the value of `"legacy"` is used as a no-op keeping the original defaults set for that use.

Reply:
https://github.com/sparkletown/sparkle/pull/1870#discussion_r674119797

Reply:
> is used as a no-op keeping the original defaults set 

Will resolve it here and leave the linked comment the place to continue